operations/configuration_management/Exporting_and_Importing_CFS_Data.md

```diff
@@ -96,6 +96,13 @@ criteria listed above.
       /usr/share/doc/csm/scripts/operations/configuration/import_cfs_data.sh /tmp/cfs-export-20230410170613-Tg0nap.tgz
       ```
 
+   - If there are running or pending CFS sessions then the import will fail. In such cases, the import can be forced by using the
```

Comment:
I would change how this is organized. The other two bullet options are explaining the two scenarios in which CFS data will typically be imported. The new argument you are adding really applies to both of those two bullet points, instead of being a third alternative to them.

I would instead add something to the "Modify the following example commands" paragraph. Maybe just add another sentence like: 

```
The import tool will abort if it detects incomplete CFS sessions which could potentially be impacted
by the import; to override this behavior and import anyway, the `--ignore-running-sessions` argument
may be added when invoking the import script.
```